pkg/legacy/tarball/write_test.go

```diff
@@ -357,6 +357,55 @@ func TestMultiWriteNoHistory(t *testing.T) {
 	}
 }
 
+func TestMultiWriteHistoryEmptyLayers(t *testing.T) {
+	// Make a random image with 2 layers.
```

Comment:
Does the number of layers in this image have to correspond to the number of non-empty history items below? If so, document that or make them both refer to the same variable so it's documented in code too.